packages/api/src/tasks/checkAlerts.ts

```diff
@@ -127,54 +210,80 @@ const buildEventSlackMessage = ({
 
 const fireChannelEvent = async ({
```

Comment:
Just realizing.. why didn't we call this `fireChannelAlert`?

Reply:
dont recall why. I guess its because its firing webhook 'even' ? 🤷